package.json

```diff
@@ -17,30 +17,35 @@
         "url": "https://github.com/Azure/monaco-kusto"
     },
     "scripts": {
-        "compile": "gulp compile",
-        "watch": "gulp watch",
-        "prepublish": "gulp release",
-        "test": "jest"
+        "copy_runtime_deps_to_out": "mcopy ./node_modules/@kusto/language-service/kusto.javascript.client.min.js ./out/vs/language/kusto/kusto.javascript.client.min.js && mcopy ./node_modules/@kusto/language-service-next/Kusto.Language.Bridge.min.js ./out/vs/language/kusto/Kusto.Language.Bridge.min.js && mcopy ./node_modules/@kusto/language-service/bridge.min.js ./out/vs/language/kusto/bridge.min.js && node ./node_modules/copy-dir-cli/bin/copy ./node_modules/monaco-editor-core/dev/vs ./out/vs && node ./node_modules/copy-dir-cli/bin/copy ./node_modules/monaco-editor-core/dev/vs ./out/vs",
+        "compile": "tsc -p ./src/tsconfig.json && tsc -p ./src/tsconfig.esm.json",
+        "watch": "echo dev > test/mode.txt && tsc -p ./src/tsconfig.watch.json && npm run copy_runtime_deps_to_out && concurrently \"live-server ./ \" \"tsc -w -p ./src/tsconfig.watch.json \"",
+        "test_release": "echo release > test/mode.txt && http-server -c-1 -p 8080 ./ -o index.html",
```

Comment:
looks like http-server is not listed as a dependency. when running the release task i got an error about it.

Reply:
package added